addons/survey/models/survey_survey.py

```diff
@@ -114,6 +114,8 @@ def default_get(self, fields_list):
         ('scoring_without_answers', 'Scoring without answers at the end')],
         string="Scoring", required=True, store=True, readonly=False, compute="_compute_scoring_type", precompute=True)
     scoring_success_min = fields.Float('Required Score (%)', default=80.0)
+    show_graph = fields.Boolean(string='Show Scoring Graph', default=True,
```

Comment:
If it's related to scoring then it should be prefixed like the related fields.

"scoring_show_graph" ?